src/js/UI.js

```diff
@@ -146,7 +146,6 @@ export default class UI {
     }
 }
 
-Split(['#left_pane', '#right_pane'], {
-    gutterSize: 8,
-    cursor: 'col-resize'
-  })
\ No newline at end of file
+if (window.innerWidth > 960) {
```

Comment:
This value should be somewhere configurable, and be kept in sync to what's in the media-queries. To easy to mistakenly change it, and do not update the queries, or the other way around.